### DISCUSSION TITLE
Notifications seems to not work

### DISCUSSION THREAD

**Discussion by BooterMan98** [community/community#28058] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.74.0 (74357630)
Device: iPhone 13 mini (iOS 15.6)
</details>

Notifications of new issues, discussions or pull request of any repo of my own does not seem trigger sound or anything on my phone. Since I got the app at launch I expected it to eventually notify me of things that happens on my repo, but it has never been the case. Instead I get notified by email or by colleagues. The app has never actually put a thing on the notifications, it's always empty.

This issues was also present on my other phones

**Comment by stevepopovich** [comment]
Hi @BooterMan98. So I assume you are talking about push notifications and not the notifications screen within the app. Unless you denied permissions, you should get notifications for things like review comments, approvals, mentions etc. 

You can go to `Profile` -> Settings -> Configure notifications and see your settings for push notifications types. Double check that they are what you expect. Thanks for reaching out!

**Reply by BooterMan98** [reply]
Yeah, push notifications. Though your response got through and got notified, a recent response  to an issue on one of my repos did not. I'm not assigned to it though, that seems to be the problem based on notification settings. It would be better if push notifications are on par with mail notifications, current settings seems very limited to me.

**Reply by stevepopovich** [reply]
Gotcha. I will file a feature request to expand our notifications capabilities. Thanks!
